src/main/resources/view/TaskListCard.fxml

```diff
@@ -8,9 +8,9 @@
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
-<HBox id="taskCardPane" fx:id="taskCardPane" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
+<HBox id="taskCardPane" fx:id="taskCardPane" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
     <stylesheets>
-        <URL value="@TaskCardView.css" />
+        <URL value="@../css/TaskCardView.css" />
```

Comment:
Thanks for classifying the css files together. Was starting to get a little messy.